static/app/components/events/eventCause.tsx

```diff
@@ -20,13 +23,28 @@ interface Props {
 }
 
 function EventCause({group, event, project}: Props) {
+  const organization = useOrganization();
   const [isExpanded, setIsExpanded] = useState(false);
-  const {committers} = useCommitters({
+  const {committers, fetching} = useCommitters({
     group,
     eventId: event.id,
     projectSlug: project.slug,
   });
 
+  useEffectAfterFirstRender(() => {
+    if (fetching) {
+      return;
+    }
+
+    trackAdvancedAnalyticsEvent('issue_details.suspect_commits', {
+      organization,
+      count: committers.length,
+      project_id: parseInt(project.id as string, 10),
+      group_id: parseInt(group!.id, 10),
+      issue_category: group?.issueCategory ?? IssueCategory.ERROR,
+    });
+  }, [organization, fetching, committers.length, project.id, group]);
```

Comment:
A bit worried about `group` causing this to fire off more than it should but if it doesn't seem fine